### PR TITLE
Hooli style popup

### PR DESCRIPTION
**Pull request by andreymashko7**
### Hooli style popup

[Demo](https://andreymashko7.github.io/hooli-style-popup/) |
[Code base](https://github.com/andreymashko7/hooli-style-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by andreymashko7** [comment]
changed html structure according to task, added some mobile styles

**Review by andreymashko7 — COMMENTED** [review]
changed html structure according to task, added some mobile styles

**Review by andreymashko7 — COMMENTED** [review]
changed html structure according to task, added some mobile styles

File: submissions/AndriyM/hooli-style-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="./styles.css" />
+        <title>Document</title>
```
Comment (A-Ostrovnyy) [inline submissions/AndriyM/hooli-style-popup/index.html:8]: Add propper page title

File: submissions/AndriyM/hooli-style-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        <div class="wrapper">
+            <header class="header">
+                <div class="container">
+                    <nav class="nav">
+                        <ul class="nav__list">
+                            <li class="nav__item">
+                                <input class="popup__input" type="checkbox" id="check" />
+                                <label for="check" class="popup__label">
+                                    <img src="./img/pngwing.png" width="35" height="35" class="nav__img-link" alt="popup open button" />
+                                </label>
```
Comment (A-Ostrovnyy) [inline submissions/AndriyM/hooli-style-popup/index.html:20]: Fix position of focused elements
image

File: submissions/AndriyM/hooli-style-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                                <ul class="popup__list">
+                                    <li class="popup__item">
+                                        <a href="#" class="popup__link"><img src="./img/blogs.png" width="90" height="60" class="popup__link-img" alt="blog link" /><span class="popup__text">Blog</span></a>
```
Comment (A-Ostrovnyy) [inline submissions/AndriyM/hooli-style-popup/index.html:23]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/AndriyM/hooli-style-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                            </li>
+                            <li class="nav__item">
+                                <a href="#" class="nav__link"><img src="./img/3425.jpg" width="35" height="35" class="nav__img-link avatar" alt="avatar picture" /></a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+            </header>
+            <main class="main"></main>
+            <footer class="footer"></footer>
```
Comment (A-Ostrovnyy) [inline submissions/AndriyM/hooli-style-popup/index.html:77]: Avoid unused tags

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Comment by andreymashko7** [comment]
corrected the last comments

File: submissions/AndriyM/hooli-style-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="./styles.css" />
+        <title>Html-css-popup</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <header class="header">
+                <div class="container">
+                    <nav class="nav">
+                        <ul class="nav__list">
```
Comment (A-Ostrovnyy) [inline submissions/AndriyM/hooli-style-popup/index.html:15]: Avoid shifting sibling elements on hover or focus. [Watch video](https://share.vidyard.com/watch/1hbFDzeSZpV3VqPfdwQCgc?)

File: submissions/AndriyM/hooli-style-popup/index.html [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+                                <a href="#" class="nav__link">Images</a>
+                            </li>
+                            <li class="nav__item">
+                                <a href="#" class="nav__link"
+                                    ><img
+                                        src="./img/bell-1.png"
+                                        width="35"
+                                        height="35"
+                                        class="nav__img-link"
+                                        alt="button to subscribe to alerts"
```
Comment (A-Ostrovnyy) [inline submissions/AndriyM/hooli-style-popup/index.html:199]: image

File: submissions/AndriyM/hooli-style-popup/index.html [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+                                /></a>
+                            </li>
+                            <li class="nav__item">
+                                <a href="#" class="nav__link"
+                                    ><img
+                                        src="./img/3425.jpg"
+                                        width="35"
+                                        height="35"
+                                        class="nav__img-link avatar"
+                                        alt=""
```
Comment (A-Ostrovnyy) [inline submissions/AndriyM/hooli-style-popup/index.html:209]: Why is this icon without an `alt` value, but the rest icon have an `alt` value?

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

**Comment by andreymashko7** [comment]
I hope it's better now.

File: submissions/AndriyM/hooli-style-popup/index.html [hunk 8]
```diff
@@ -0,0 +1,219 @@
...
+                                    type="checkbox"
+                                    id="check"
+                                />
+                                <label for="check" class="popup__label">
+                                    <img
+                                        src="./img/pngwing.png"
+                                        width="35"
+                                        height="35"
+                                        class="nav__img-link"
+                                        alt="show menu-popup links"
```
Comment (A-Ostrovnyy) [inline submissions/AndriyM/hooli-style-popup/index.html:28]: For decorative images implemented with `img` tag, `alt` attribute value should be an empty value.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

**Comment by andreymashko7** [comment]
fixed alt

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**